python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -30,16 +31,16 @@
 )
 
 
-def _check_groupby_supported(func):
+def _check_groupby_optimized(func):
     """
     Decorator for dask-cudf's groupby methods that returns the dask-cudf
-    method if the groupby object is supported, otherwise reverting to the
-    upstream Dask method
+    optimized method if the groupby object is supported, otherwise
+    reverting to the upstream Dask method
     """
 
     def wrapper(*args, **kwargs):
```

Comment:
Can we `@functools.wraps(func)` this decorator so that docstring/arglist is correct in the interpreter?